### ISSUE TITLE
Uncaught Error: Undefined screen line when clipping screen position

### ISSUE THREAD

**Issue by rugk** [olmokramer/atom-minimap-codeglance#19]
Happens when the mouse is moved over the minimap.
Disabling this package fixes it.

**Atom Version**: 1.1.0
**System**: Windows 7
**Thrown From**: [aligner](https://github.com/adrianlee44/atom-aligner) package, v0.17.1
### Stack Trace

Uncaught Error: Undefined screen line when clipping screen position

```
At C:\Users\%username%\AppData\Local\atom\app-1.1.0\resources\app.asar\src\display-buffer.js:778

Error: Undefined screen line when clipping screen position
  at DisplayBuffer.module.exports.DisplayBuffer.clipScreenPosition (C:\Users\%username%\AppData\Local\atom\app-1.1.0\resources\app.asar\src\display-buffer.js:766:15)
  at DisplayBuffer.module.exports.DisplayBuffer.bufferPositionForScreenPosition (C:\Users\%username%\AppData\Local\atom\app-1.1.0\resources\app.asar\src\display-buffer.js:718:20)
  at TextEditor.module.exports.TextEditor.bufferPositionForScreenPosition (C:\Users\%username%\AppData\Local\atom\app-1.1.0\resources\app.asar\src\text-editor.js:1139:33)
  at Object.scrollToMinimapOffset (D:\Users\%username%\.atom\packages\minimap-codeglance\lib\minimap-codeglance.js:145:62)
  at atom-text-editor-minimap.<anonymous> (D:\Users\%username%\.atom\packages\minimap-codeglance\lib\minimap-codeglance.js:93:14)

```
### Commands

```
     -0:33.1.0 editor:consolidate-selections (atom-text-editor.editor.is-focused)
     -0:33.1.0 core:cancel (atom-text-editor.editor.is-focused)
     -0:32.9.0 editor:consolidate-selections (atom-text-editor.editor.is-focused)
     -0:32.9.0 core:cancel (atom-text-editor.editor.is-focused)
     -0:32.6.0 editor:consolidate-selections (atom-text-editor.editor.is-focused)
     -0:32.6.0 core:cancel (atom-text-editor.editor.is-focused)
     -0:30.3.0 editor:consolidate-selections (atom-text-editor.editor.is-focused)
     -0:30.3.0 core:cancel (atom-text-editor.editor.is-focused)
     -0:29.6.0 editor:consolidate-selections (atom-text-editor.editor.is-focused)
     -0:29.6.0 core:cancel (atom-text-editor.editor.is-focused)
     -0:27.5.0 editor:consolidate-selections (atom-text-editor.editor.is-focused)
     -0:27.5.0 core:cancel (atom-text-editor.editor.is-focused)
     -0:25.3.0 editor:consolidate-selections (atom-text-editor.editor.is-focused)
     -0:25.3.0 core:cancel (atom-text-editor.editor.is-focused)
     -0:24 editor:consolidate-selections (atom-text-editor.editor.is-focused)
     -0:24 core:cancel (atom-text-editor.editor.is-focused)
```
### Config

``` json
{
  "core": {
    "disabledPackages": [
      "autocomplete-emojis",
      "metrics",
      "package-cop",
      "tree-view-breadcrumb",
      "foldername-tabs",
      "markdown-table-formatter",
      "minimap-hide"
    ],
    "themes": [
      "atom-dark-ui",
      "one-dark-syntax"
    ]
  }
}
```
### Installed Packages

``` coffee
# User
aligner, v0.17.1
aligner-php, v1.0.0
aligner-scss, v1.1.0
autocomplete-php, v0.3.7
autohide-tree-view, v0.25.1
color-picker, v2.0.13
color-tabs, v0.1.8
compare-files, v0.6.2
editor-stats, v0.17.0
figlet, v0.5.2
file-icons, v1.6.11
highlight-column, v0.5.0
highlight-selected, v0.10.1
keyboard-localization, v1.4.10
language-apache, v1.3.0
language-autoit, v0.2.1
language-batch, v0.3.3
language-ini, v1.14.0
language-nginx, v0.6.1
language-pascal, v0.12.0
line-count-status, v0.0.3
line-diff-details, v1.1.3
linter, v1.10.0
linter-alex, v1.5.0
linter-htmlhint, v0.2.1
linter-php, v1.0.5
markdown-preview-opener, v0.1.1
minimap, v4.16.0
minimap-bookmarks, v0.1.0
minimap-codeglance, v0.4.2
minimap-find-and-replace, v4.3.2
minimap-git-diff, v4.1.8
minimap-highlight-selected, v4.3.1
minimap-linter, v1.0.0
minimap-pigments, v0.1.7
minimap-selection, v4.3.1
opened-files, v0.2.5
php-cs-fixer, v2.5.1
pigments, v0.18.1
switcheroo, v1.1.0
tablr, v0.6.1
tabs-to-spaces, v0.11.1
tidy-markdown, v1.0.0
tree-view-background, v0.4.0

# Dev
No dev packages
```

**Comment by ieltxualganaras** [comment]
:+1: this also happens to me on linux.

**Comment by olmokramer** [comment]
You seem to be on v0.4.2 of minimap-codeglance, while the most recent version is v0.4.4 (soon to be 0.4.5). https://github.com/olmokramer/atom-minimap-codeglance/commit/defd249a47729f7b6fcbc6c44367786c4a1b94ef should have fixed this, can you please confirm that? (after I've released v0.4.5 because there seem to be some other issues with v0.4.4 and Atom v1.1.0)

**Comment by rugk** [comment]
Updated and it works with v0.4.5. :+1:
